src/main/java/com/direwolf20/buildinggadgets/common/building/PlacementTarget.java

```diff
@@ -0,0 +1,29 @@
+package com.direwolf20.buildinggadgets.common.building;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+public final class PlacementTarget {
+
+    private final IBlockState state;
+    private final BlockPos pos;
+
+    public PlacementTarget(IBlockState state, BlockPos pos) {
+        this.state = state;
+        this.pos = pos;
+    }
+
+    public IBlockState getBlockState() {
+        return state;
+    }
+
+    public BlockPos getPos() {
+        return pos;
+    }
+
+    public void placeIn(World world) {
+        world.setBlockState(pos, state);
+    }
+
+}
```

Comment:
This class isn't used. Are there plans to use it in the future?

Reply:
Yes, @MajorTuvok requested it for a replace of `BlockMap`